src/evaluation/see_swap.h

```diff
@@ -81,11 +81,12 @@ class SeeSwap {
         /* piece that will track the scoring of next piece */
         Piece nextPiece = move.piece();
 
-        if (const auto initialPiece = board.getPieceAtSquare(move.toSquare())) {
+        Player player = board.player;
+
+        if (const auto initialPiece = board.getTargetAtSquare(move.toSquare(), board.player)) {
```

Comment:
Assume you meant this given you moved the declaration above? :)
```suggestion
        if (const auto initialPiece = board.getTargetAtSquare(move.toSquare(), player)) {
```